lib/kafka/concerns/topic.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'active_support/concern'
+
+module Kafka
+  module Topic
+    extend ActiveSupport::Concern
```

Comment:
The way you're currently using the module you don't actually need to extend `ActiveSupport::Concern`. Just including the module would give access to instance methods defined here.